thrift/thrift.bzl

```diff
@@ -160,6 +165,9 @@ thrift_library = rule(
         "absolute_prefix": attr.string(default = '', mandatory = False),
         "absolute_prefixes": attr.string_list(),
         # This is a list of JARs which only contain Thrift files
+        # these files will NOT  be compiled as part of the current target,
+        # but the current thrifts do include these. It should also include
```

Comment:
so, I kept it segregated, but now we compile the jars. This allows you to do import style of remote items by using a target with only an external_jar. mixing them with the srcs ultimately seemed confusing to me since I don't think we want to rezip them into the output or apply the prefix rules to them. What do you think about this?

It has the side effect, I think, of making current uses just work, although slower if they include a giant jar over and over.